File: FullStack_Java_Roadmap/src/fundamentals/day06/Problem2.java
package fundamentals.day06;

public class Problem2 {
    public static void main(String[] args) {
        int[] numbers = {10, 15, 20, 25, 30, 35};
        int count = 0;

        for (int number : numbers) {
            if (number % 2 == 0) {
                count++;
            }
        }

        System.out.println("Even Numbers: " + count);
    }
}

File: python/fundamentals/day-06/Problem2.py
numbers = [10, 15, 20, 25, 30, 35]

# Standard loop approach
count = 0
for number in numbers:
    if number % 2 == 0:
        count += 1

print(f"Even Numbers: {count}")

# One-line Pythonic alternative:
# count = sum(1 for num in numbers if num % 2 == 0)